container-guide/concepts/containers-tools.xml

```diff
@@ -0,0 +1,207 @@
+      images and image repositories. skopeo can be used to inspect containers
+      and repositories on remote and local container registries. skopeo can
+      copy container images between different storage back-ends. skopeo is part
+      of the <literal>Basesystem Module</literal> of &sles;.
+    </para>
+  </section>
+  <section xml:id="sec-tools-helm">
+    <title>Helm</title>
+    <para>
+      <link xlink:href="https://helm.sh">Helm</link> is the Kubernetes package
+      manager, and it is the de-facto standard for deploying containerized
+      applications on Kubernetes clusters using charts. Helm can be used to
+      install, update and remove applications in a single or multiple
+      containers. It can also handle the associated resources, such as
+      configuration, storage volumes, etc. For instance, it is used for
+      instance to deploy the &rmt; server (see
+      <link xlink:href="https://documentation.suse.com/sles/15-SP4/single-html/SLES-rmt/index.html#sec-rmt-deploy-kubernetes">&rmt;
+      documentation</link> for more information).
+    </para>
+  </section>
+  <section xml:id="sec-tools-distribution">
+    <title>Distribution</title>
+    <para>
+      <link xlink:href="https://github.com/distribution/distribution">Distribution</link>
+      is an open-source registry implementation for storing and distributing
+      container images using the OCI Distribution Specification. It provides a
+      simple, secure and scalable base for building a large scale registry
+      solution or running a simple private registry. Distribution can also
+      mirror &dhub; but
+      <link xlink:href="https://docs.docker.com/registry/recipes/mirror/#gotcha">not
+      any other private registry</link>.
+    </para>
+  </section>
+  <section xml:id="sec-tools-obs">
+    <title>Open Build Service</title>
+    <para>
+      The <link xlink:href="http://openbuildservice.org">Open Build Service
+      (OBS)</link> provides free infrastructure for building and storing RPM
+      packages including different container formats. The
+      <link xlink:href="https://registry.opensuse.org">OBS Container
+      Registry</link> provides a detailed listing of all container images built
+      by the OBS, complete with commands for pulling the images into your local
+      environment. The OBS
+      <link xlink:href="https://build.opensuse.org/image_templates">openSUSEcontainer
+      image templates</link> can be modified to specific needs, which offers
+      the easiest way to create your own container branch. Container images can
+      be built with &docker; tools from an existing image using a
+      <filename>Dockerfile</filename>. Alternatively, images can be built from
+      scratch using the &kiwi; image-building solution.
+    </para>
+    <para>
+      Instructions on how to build images on OBS can be found
+      <link xlink:href="https://openbuildservice.org/2018/05/09/container-building-and-distribution/">here</link>.
```

Comment:
Uh-oh, "here" is such an awkward word. "Here" is very relative and confusing. On my side, nothing is "here". Looking from my side, the remote URL is "there". :wink: 

Joking aside, I would recommend to describe the "here" part. In this case it's sounds like blog article. Therefore:

```suggestion
      <link xlink:href="https://openbuildservice.org/2018/05/09/container-building-and-distribution/">in this blog article</link>.
```